tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -178,5 +178,7 @@ def _print(self, result: Mapping[str, Any]) -> Any:
         return result["data"]
 
     @command
-    def set_viewport(self, context: str, viewport: Optional[Mapping[str, Any]] = None) -> Mapping[str, Any]:
+    def set_viewport(self, context: str, viewport: Optional[Mapping[str, Any]] | 'undefined' = None) -> Mapping[str, Any]:
```

Comment:
Can we use typing to define an undefined type? Or could we define a class for it?

Reply:
I think this would need agreement in all of WPT.